### DISCUSSION TITLE
Is there has a function can cut image blank area?

### DISCUSSION THREAD

**Discussion by df257** [SixLabors/ImageSharp#1749] [Q&A] [answered]
like photoshop
![image](https://user-images.githubusercontent.com/7835444/130783789-ef83dab2-82c4-4bde-8389-56303f2cd4d9.png)

**Comment by antonfirsov** [comment]
@df257 can you elaborate more on what you mean? It's not clear from the screenshot, especially if it's not in English.

**Reply by df257** [reply]
see this video
https://user-images.githubusercontent.com/7835444/130895415-fc48c39e-30a7-4c67-a7d7-c94dbdf978ca.mp4

**Comment by JimBobSquarePants** [comment, marked as answer]
From the video it looks like you're looking for [EntropyCrop](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Processing.EntropyCropExtensions.html#methods)
